### ISSUE TITLE
Unhandled exception

### ISSUE THREAD

**Issue by speedyman09** [LBPUnion/PLRPC#4]
**Describe the bug**
Opening the program results in error message: Unhandled exception. System.IndexOutOfRangeException: Index was outside the bounds of the array. 
I am using the .NET 7 Runtime. 

**To Reproduce**
1. Open the program

**Screenshots**
![Screenshot 2023-04-14 125002](https://user-images.githubusercontent.com/17880243/232016494-1c1c86c7-60d9-4a1b-873e-3659a28b8324.png)

**Desktop**
 - OS: Windows 10
 - v0.1.1. 
 - Commit **https://github.com/LBPUnion/PLRPC/commit/b26ded5083732a736eb15cfbc97653656b58f658**

**Comment by sudokoko** [comment]
Thank you for this. I believe I know the cause of this, however can you post the entire command you entered to start the client? Much appreciated :)

**Comment by speedyman09** [comment]
All I'm doing is just writing PLRPC.exe in the program directory. also i created a config file using PLRPC.exe --config and put my lighthouse name into it.

**Comment by sudokoko** [comment]
Yup, just replicated the issue. In order to actually use that config, you have to always run the program with --config. However the program just crashes when the flag isn't there because of a mistake on my part, whoops!

**Comment by speedyman09** [comment]
Oh, alright. thanks for your help

**Comment by sudokoko** [comment]
No problem. I'll get a patch rolled out here soon.

**Comment by sudokoko** [comment]
Fixed in 53656193bc63f12353d1c758ab6b3b9aad58e987.
